### DISCUSSION TITLE
Multiple entity as heater

### DISCUSSION THREAD

**Discussion by M-ERiMiEs** [ScratMan/HASmartThermostat#217] [Q&A]
I tried to look but couldnt find out does Smart thermostat support multiple entity as heater? I have underfloor water heating and its controlled by individual valves on every circuit and I have some rooms that has up to 6 circuits.

**Comment by ScratMan** [comment]
It will support multiple entities in the next release. For the moment you may use groups

**Comment by ScratMan** [comment]
Added in [v2024.4.0-beta1](https://github.com/ScratMan/HASmartThermostat/releases/tag/2024.4.0-beta1)
